olive/resource_path.py

```diff
@@ -40,7 +40,6 @@ class ResourceType(CaseInsensitiveEnum):
 LOCAL_RESOURCE_TYPES = (ResourceType.LocalFile, ResourceType.LocalFolder)
 AZUREML_RESOURCE_TYPES = (
     ResourceType.AzureMLModel,
-    ResourceType.AzureMLRegistryModel,
```

Comment:
is there a reason for this change?

Reply:
since registrymodel does not require workspace information?

Reply:
Yes, a couple days ago I added this for shared cache: https://github.com/microsoft/Olive/pull/1417. Then I realize it is used for https://github.com/microsoft/Olive/blob/main/olive/workflows/run/config.py#L314 as well. So instead of updating config file, i reverted my changes about 1417 here.